pcap-openvizsla.c

```diff
@@ -0,0 +1,199 @@
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "pcap-int.h"
+#include "pcap-openvizsla.h"
+
+#include <ov.h>
+
+#define OPENVIZSLA_IFACE "openvizsla"
+
+struct pcap_openvizsla {
+	enum ov_usb_speed usb_speed;
+	struct ov_device* ov;
+};
+
+struct pcap_openvizsla_read {
+	pcap_handler callback;
+	u_char *user;
+};
+
+static void openvizsla_read_cb(struct ov_packet* packet, void* data) {
+	struct pcap_openvizsla_read* r = (struct pcap_openvizsla_read*)data;
+
+	struct pcap_pkthdr pkth;
+	pkth.caplen = ov_to_host_16(packet->size) + sizeof(struct ov_packet);
+	pkth.len = pkth.caplen;
+	pkth.ts.tv_sec = 0;
```

Comment:
Can no time stamp be supplied?

Reply:
The device provides 24-bit timestamp with precision of `1/60 usec`. Unfortunately, this value wraps around many times during the capture. What would be preferred way to provide best possible timestamp (combining 24-bit timestamp and the host wall clock)?

Reply:
So the 24-bit timestamp goes back to zero about 3.58 times per second, is that correct? (If yes, then just a combination of UNIX time and the 24-bit timestamp does not tell the exact time when a packet was received, especially at less then 3.58 packets per second, it needs to be something else.)

Is there any way to keep the 24-bit timestamp in sync with the host clock, such as sending a pulse to the device every second or a fraction of a second?

Reply:
Correct. I think that the simplest way would be to use UNIX host receive timestamp and keep the counter as is.